### PR TITLE
Plot SSL_WrapperPacket in Jupyter Notebook using Bokeh

### LINKED ISSUE #1901: Plot world state from SSL_WrapperPacket using Bokeh
### Acceptance criteria

- [ ] Item A
- [ ] Item B
- [ ] Item C

### PR DESCRIPTION
**Pull request by jordanzliu**
### Description

pull in bokeh as a dependency and develop a set of helper functions to plot the data in SSL_WrapperPacket using bokeh

### Testing Done

image

### Resolved Issues

Resolves #1901 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/python_tools/plotting/ssl_wrapper_plot_helpers.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
+import numpy as np
```
Comment (akhilveeraghanta) [inline src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: nit: you can probably just call this file `plot_ssl_wrapper.py`

File: src/python_tools/plotting/ssl_wrapper_plot_helpers.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+def plot_circular_section(
+    fig: figure,
+    center_x: float,
+    center_y: float,
+    radius: float,
+    start_angle: float,
+    end_angle: float,
+):
+    # decide how many samples to take along the arc, a full circle will be a 36-gon
```
Comment (akhilveeraghanta) [inline src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: this could maybe use a sphinx style doc

**Review by akhilveeraghanta — COMMENTED** [review]
looks great :fire: , just 2 nits from my end

File: src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "disturbed-butterfly",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from software.proto.messages_robocup_ssl_wrapper_pb2 import SSL_WrapperPacket\n",
+    "from python_tools.proto_log import ProtoLog\n",
+    "\n",
+    "wrapper_proto_log = ProtoLog(\"/home/jordan/log_ssl_wrapper_filtered_world_test/SensorFusion_SSL_WrapperPacket\", \n",
```
Comment (jonathanlew) [inline src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb:13]: I obviously don't have this directory on my computer. Can the ProtoLog function fail fast when the directory doesn't exist?

**Review by jonathanlew — CHANGES_REQUESTED** [review]
We could use more documentation for new members, but that's probably a larger guide in  getting_started.md

File: src/python_tools/plotting/ssl_wrapper_plot_helpers.py [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    SSL_FieldLineSegment,
+)
+from software.proto.messages_robocup_ssl_detection_pb2 import (
+    SSL_DetectionBall,
+    SSL_DetectionRobot,
+)
+from software.proto.messages_robocup_ssl_wrapper_pb2 import SSL_WrapperPacket
+from typing import Iterable
+
+ROBOT_MAX_RADIUS = 0.09
```
Comment (itsarune) [inline src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: I was wondering whether it matters that these constants match the constants defined in the other parts of the code base (ROBOT_MAX_RADIUS and BALL-RADIUS)

File: src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb (hunk 3 above)
Comment (jordanzliu) [reply src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb:13]: Opened #1909

File: src/python_tools/plotting/ssl_wrapper_plot_helpers.py (hunk 4 above)
Comment (jonathanlew) [reply src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: this is good point, can we use python-cpp bindings to keep them consistent?

Comment (jordanzliu) [reply src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: the bindings are too much of a pain to set up just for this

Comment (jordanzliu) [reply src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: the ball radius is currently an arbitrary value but the ROBOT_MAX_RADIUS is copypasted from the cpp code. It depends on what users use this for, for most use cases I think a "good enough" value works.

File: src/python_tools/plotting/ssl_wrapper_plot_helpers.py (hunk 2 above)
Comment (jordanzliu) [reply src/python_tools/plotting/ssl_wrapper_plot_helpers.py]: added comments

**Review by jonathanlew — APPROVED** [review]
LGTM pending other comments

**Review by akhilveeraghanta — APPROVED** [review]
:fire: can't wait to use it

**Pull request merged**